python/shipDet_conf.py

```diff
@@ -187,42 +187,48 @@ def configure(run,ship_geo):
-   taumuondetector.SetPillarDimensions(ship_geo.tauMudet.PillarX,ship_geo.tauMudet.PillarY, ship_geo.tauMudet.PillarZ)
-   detectorList.append(taumuondetector)
-   if ship_geo.nuTauTargetDesign==3:
-    if hasattr(ship_geo.tauMudet, "deltax"): #now replaced with veto taggers
-     taumuondetector.SetRpcDimDifferences(ship_geo.tauMudet.deltax, ship_geo.tauMudet.deltay)
-   if ship_geo.nuTauTargetDesign<3:
-    taumuondetector.SetReturnYokeDimensions(ship_geo.tauMudet.XRyoke,ship_geo.tauMudet.YRyoke, ship_geo.tauMudet.ZRyoke)
-    taumuondetector.SetSmallerYokeDimensions(ship_geo.tauMudet.XRyoke_s,ship_geo.tauMudet.YRyoke_s, ship_geo.tauMudet.ZRyoke_s)
-    taumuondetector.SetZDimensionArm(ship_geo.tauMudet.ZArm)
-    taumuondetector.SetGapDownstream(ship_geo.tauMudet.GapD)
-    taumuondetector.SetGapMiddle(ship_geo.tauMudet.GapM)
-    taumuondetector.SetMagneticField(ship_geo.tauMudet.B)
-    taumuondetector.SetCoilParameters(ship_geo.tauMudet.CoilH, ship_geo.tauMudet.CoilW, ship_geo.tauMudet.N, ship_geo.tauMudet.CoilG)
+    taumuondetector.SetRpcDimensions(ship_geo.tauMudet.XRpc,ship_geo.tauMudet.YRpc, ship_geo.tauMudet.ZRpc)
+    taumuondetector.SetRpcGasDimensions(ship_geo.tauMudet.XGas,ship_geo.tauMudet.YGas, ship_geo.tauMudet.ZGas)
+    taumuondetector.SetRpcStripDimensions(ship_geo.tauMudet.XStrip,ship_geo.tauMudet.YStrip, ship_geo.tauMudet.ZStrip)
+    taumuondetector.SetRpcElectrodeDimensions(ship_geo.tauMudet.XEle,ship_geo.tauMudet.YEle, ship_geo.tauMudet.ZEle)
+    taumuondetector.SetRpcPETDimensions(ship_geo.tauMudet.XPet,ship_geo.tauMudet.YPet, ship_geo.tauMudet.ZPet)
+    taumuondetector.SetNRpcInArm(ship_geo.tauMudet.NRpc)
+    if( ship_geo.nuTauTargetDesign ==3):
+     taumuondetector.SetUpperCoverDimensions(ship_geo.tauMudet.XCov, ship_geo.tauMudet.YCov, ship_geo.tauMudet.ZCov)
+     taumuondetector.SetLateralCoverDimensions(ship_geo.tauMudet.XLateral, ship_geo.tauMudet.YLateral, ship_geo.tauMudet.ZLateral)
+     taumuondetector.SetCrossDimensions(ship_geo.tauMudet.XCross, ship_geo.tauMudet.YCross, ship_geo.tauMudet.ZCross, ship_geo.tauMudet.WidthArm)
+     taumuondetector.SetRpcOuterDimensions(ship_geo.tauMudet.XRpc_outer, ship_geo.tauMudet.YRpc_outer, ship_geo.tauMudet.ZRpc_outer)
+     taumuondetector.SetRpcInnerDimensions(ship_geo.tauMudet.XRpc_inner, ship_geo.tauMudet.YRpc_inner, ship_geo.tauMudet.ZRpc_inner)
+     taumuondetector.SetRpcGapDimensions(ship_geo.tauMudet.XRpcGap, ship_geo.tauMudet.YRpcGap, ship_geo.tauMudet.ZRpcGap)
+     taumuondetector.SetPillarDimensions(ship_geo.tauMudet.PillarX,ship_geo.tauMudet.PillarY, ship_geo.tauMudet.PillarZ)   
+    if ship_geo.nuTauTargetDesign==3:
+     if hasattr(ship_geo.tauMudet, "deltax"): #now replaced with veto taggers
+      taumuondetector.SetRpcDimDifferences(ship_geo.tauMudet.deltax, ship_geo.tauMudet.deltay)
+    if ship_geo.nuTauTargetDesign<3:
+     taumuondetector.SetReturnYokeDimensions(ship_geo.tauMudet.XRyoke,ship_geo.tauMudet.YRyoke, ship_geo.tauMudet.ZRyoke)
```

Comment:
Not sure what's going on with the white space here...

Reply:
TODO for later: we need to fix the white space on nearly all python files...

Reply:
In this case the increase of white space is intentional, since I have added another if before:
if( ship_geo.nuTauTargetDesign<4)

Reply:
Ah, I see. OK, but we really need to start using some normal python white space convention (2 or 4 spaces) to make this kind of thing readable. Probably need to change this one file at a time though...